python/lsst/verify/__init__.py

```diff
@@ -27,6 +27,7 @@
     __version__ = "unknown"
 
 from .errors import *
+from .extract_metricvalues import *
```

Comment:
If the goal is just to add documentation, why not add an appropriate `automodapi::` directive? It seems a bit strange for this to be living in `verify`, when other things (e.g., `verify.tasks`) don't.

Reply:
Also, do you need to add some extra links to the docs to clarify that the `extract_metricvalues` methods are for imitating `print_metricvalues` in a notebook? I imagine that right now the methods just appear with no context.

Reply:
Good idea: I forgot that you can just add `automodapi` to get those docs in.

Your suggestion about the extra links is a very good one, but I'm not sure where those would go. The `automodapi` block just appears on its own in the built docs; I don't think we're supposed to add any text to the module section. I can remove the `:no-main-docstr:` directive and expand the file docstring, but the dev guide says not to do that. Should I ask on #dm-docs-support for suggestions?

Reply:
As mentioned in the main comment, cross-referencing `extract_metricvalues` from the `print_metricvalues` docs seems like a good idea (since that's a likely place for anybody who wants this functionality to look).

As for `:no-main-docstr:`, it seemed like the least of the available evils, but it's true that [other modules don't use it](https://pipelines.lsst.io/v/daily/modules/lsst.verify/index.html#lsst-verify-bin-dispatchverify-module). 🤷

Reply:
Oh, see [this Slack thread](https://lsstc.slack.com/archives/C2B6DQBAL/p1656102289864929) for a discission of the rationale and consequences of the no `:no-main-docstr:` policy.

Reply:
Yeah, I'm behind on making the RFC to introduce module docstrings (i.e. removing the `no-main-docstr` policy) so you'd be able to directly reference a module using API linking syntax (i.e. single backticks).

For expedience, I advocate just dropping the `no-main-docstr` on the `automodapi` directives here as needed. When you do this, check that the module docstrings are no longer than a single sentence so we don't making the API docs harder to navigate with lots of interstitial module docs. If there's more info, either delete it, move it into a code comment, or move it elsewhere (like narrative docs in an rst page) as appropriate.

Reply:
Does it have to be a single sentence, or would a short paragraph be ok? The latter makes more sense to me (first sentence describes the module, second sentence describes and links some other pages, etc.).

Reply:
I'd recommend a single sentence, but generally keep it extremely short. A long module docstring will make hard to navigate around the Python APIs section, which is really meant to be a table of contents and not a place for actual documentation content.